rapids-cmake/cpm/versions.json

```
@@ -78,8 +78,8 @@
     },
     "rmm": {
       "version": "${rapids-cmake-version}",
-      "git_url": "https://github.com/rapidsai/rmm.git",
-      "git_tag": "branch-${version}"
+      "git_url": "https://github.com/bdice/rmm.git",
+      "git_tag": "5685e7d180b00a139f70c88389eda786ac33d9a4"
```

This needs to be reverted before merging.

```suggestion
      "git_url": "https://github.com/rapidsai/rmm.git",
      "git_tag": "branch-${version}"
```